src/border_agent/border_agent.cpp

```diff
@@ -77,6 +77,8 @@ static const char    kBorderAgentServiceType[]      = "_meshcop._udp";   ///< Bo
 static const char    kBorderAgentEpskcServiceType[] = "_meshcop-e._udp"; ///< Border agent ePSKc service
 static constexpr int kBorderAgentServiceDummyPort   = 49152;
 
+static const uint16_t kThreadVersion14 = 5; ///< Thread Version 1.4
```

Comment:
looks duplicate with the definition below
https://github.com/openthread/ot-br-posix/blob/92ac6cb66cd39aca8c70d852d504bb56fc9e2f3c/src/ncp/rcp_host.cpp#L64-L67
may consider adding a constant.hpp file under src/common?

Reply:
we already have similar definition eg. in rcp_host.cpp; it seems it requires a refactor to use the common definition. Could this be added to TODO?

Reply:
sg

Reply:
track in b/358571361. Thanks!